app/src/main/java/com/victor/adasfeed/ProfileActivity.kt

```diff
@@ -72,4 +91,13 @@ class ProfileActivity : AppCompatActivity() {
         super.onResume()
         Log.d("ciclo de vida PA", "onResume")
     }
+
+    override fun onDestroy() {
+        val intent = Intent(applicationContext, FeedActivity::class.java).apply {
+            putExtra(EXTRA_KEY, user)
+        }
+        startActivity(intent)
```

Comment:
Nice job implementing lifecycle 🚀 
But think in the scenario that you put the app on background and other apps which demand more memory to load - the activity would be detroyed, and then you would call this `startActivity` without the app is alive. We could handle datas that we want to save as "cache" for the user in lifecycles, for example, but we try to avoid these complex datas here 😄